packages/webpack-config/src/index.js

```diff
@@ -125,20 +148,38 @@ const baseConfig = {
 					{
 						loader: 'sass-loader',
 						options: {
-							sourceMap: (mode === 'development'),
+							sourceMap: shouldUseSourceMap,
 							// Set scss debug flag
 							data: `$IS_DEBUG: ${(mode === 'development')};`,
 						},
 					},
 				],
 			},
 
-			// JavaScript - babel-loader transpiles our javascript to ensure browser compatability
 			{
-				test: /\.js$/,
-				exclude: /node_modules/,
-				use: ['babel-loader'],
+				test: /\.(svg|png|jpg|ico|gif|woff|woff2|ttf|eot|doc|pdf|zip|wav|avi|txt)(\?v=\d+\.\d+\.\d+)?$/,
+				oneOf: [
```

Comment:
Unfortunately `oneOf` makes it hard to override the config with webpack-merge. Can this be split into two seperate rules, one for `[/\.svg$/, /\.gif$/, /\.jpe?g$/, /\.png$/]` and one for everything else?